quickwit-proto/proto/cluster.proto

```diff
@@ -53,6 +53,9 @@ message ListMembersRequest {
 }
 
 message ListMembersResponse {
```

Comment:
Should we keep `ListMembersResponse` or go for `ClusterState` as mentioned in a JS file?
I'm ok to not change the name now, there is another issue that suggest dumping all keys/values of each members so that we have the full view of the cluster from a given node.

Reply:
Yeah, the naming isn't great. I'd like to see how the use of this endpoint evolves before we start renaming things.

Reply:
yes exactly, so we are aligned :)